ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -106,7 +110,10 @@ - (UIView *)viewForTag:(NSNumber *)tag
 {
   UIView *view = [_reaUiManager viewForReactTag:tag];
   if (view == nil) {
-    return [_exitingViews objectForKey:tag];
+    view = [_exitingViews objectForKey:tag];
+  }
+  if (view == nil) {
+    view = [_sharedTransitionManager getTransitioningView:tag];
   }
```

Comment:
So...
this could be made more concise, with something along the lines of
```objective-c
UIView *view;
(view = [_reaUiManager viewForReactTag:tag]) || (view = [_exitingViews objectForKey:tag]) || (view = [_sharedTransitionManager getTransitioningView:tag]);
return view;
```
but I'm not sure if that's better than the chain of `if` statements.

We'd like to do
```objective-c
return [_reaUiManager viewForReactTag:tag] || [_exitingViews objectForKey:tag] || [_sharedTransitionManager getTransitioningView:tag];
```
but the short-circuiting logical operators (`||` and `&&`) return `int` values (as per the C standard), and it's illegal to cast them back to pointers (in Objective C)